src/compiler/types.ts

```diff
@@ -316,6 +316,7 @@ namespace ts {
         // Property assignments
         PropertyAssignment,
         ShorthandPropertyAssignment,
+        SpreadObjectLiteralAssignment,

```

Comment:
Actually, I'll swap the name of the existing SpreadElementExpression with this in my PR, and you can just drop this change for now.